tests/Integration/classes/module/ModuleBlockReassuranceTest.php

```diff
@@ -0,0 +1,194 @@
+<?php
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace Tests\Integration;
+
+use PrestaShop\PrestaShop\Core\Addon\Module\ModuleManagerBuilder;
+use Tests\TestCase\IntegrationTestCase;
+use Module;
+
+/**
+ * Test that module blockreassurance behavior is correct
+ *
+ * Is tested:
+ * - install process
+ * - uninstall process
+ * Tests to add:
+ * - rendering of the widget (cannot be tested now, it requires proper Smarty initialization)
+ * - rendering of thr admin form (cannot be tested now, it requires proper Smarty initialization)
+ */
+class ModuleBlockReassuranceTest extends IntegrationTestCase
+{
+    /**
+     * Flag used to control whether the module should be uninstalled at tear down after a test
+     *
+     * @var bool
+     */
+    private static $shouldUninstallAtTearDown = true;
+
+    protected function setUp()
+    {
+        $moduleLocation = _PS_MODULE_DIR_ . '/blockreassurance';
+
+        $doesModuleExistIntoTestModuleDirectory = ((file_exists($moduleLocation)) && (is_dir($moduleLocation)));
```

Comment:
You can use beforeClass and markTestSkipped :) 
https://phpunit.de/manual/6.5/en/incomplete-and-skipped-tests.html

But not sure this tests should be in the core but in the module.